.github/workflows/pitest.yml

```diff
@@ -10,6 +10,19 @@ jobs:
   pitest-annotation:
     runs-on: ubuntu-latest
     steps:
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - name: Install Groovy 4.0.3
+        run: |
+          GROOVY_LINK="https://groovy.jfrog.io/artifactory/dist-release-local/groovy-zips/apache-groovy-binary-4.0.3.zip"
+          wget -O ${{ runner.temp }}/groovy.zip $GROOVY_LINK;
+          unzip ${{ runner.temp }}/groovy.zip -d ${{ runner.temp }};
+          mv ${{ runner.temp }}/groovy-4.0.3 ${{ runner.temp }}/groovy
+          GROOVY_HOME=${{ runner.temp }}/groovy
+          echo "GROOVY_HOME=${{ runner.temp }}/groovy" >> $GITHUB_ENV
+          echo "$GROOVY_HOME/bin" >> $GITHUB_PATH
```

Comment:
Please use sdkman https://sdkman.io/sdks#groovy, let's also use a yml anchor or some other way to avoid too much copy paste.

Reply:
Done, removed the duplication, we use matrix now.